nilearn/signal.py

```diff
@@ -167,6 +168,7 @@ def _mean_of_squares(signals, n_batches=20):
     return var
 
 
+# UNUSED AND UNTESTED
 def _row_sum_of_squares(signals, n_batches=20):
```

Comment:
Should we remove it?

Reply:
Maybe first we can look at the git blame and track down the PR to see why this was added. Could it be useful?

Reply:
makes sense

Reply:
OK this is a case of PEBKAC "Rémi cannot do a proper `ctrl + f`"...

There is a test for and it is used...

Reply:
# 🙈